inc/classes/subscriber/Plugin/class-updater-api-common-subscriber.php

```diff
@@ -148,10 +148,9 @@ public function maybe_set_rocket_user_agent( $request, $url ) {
 	public function get_rocket_user_agent() {
 		$consumer_key   = $this->get_current_option( 'consumer_key' );
 		$consumer_email = $this->get_current_option( 'consumer_email' );
-		$bonus          = $this->plugin_options && $this->plugin_options->get( 'do_beta' ) ? '+' : '';
 		$php_version    = preg_replace( '@^(\d+\.\d+).*@', '\1', phpversion() );
 
-		return sprintf( 'WP-Rocket|%s%s|%s|%s|%s|%s;', $this->plugin_version, $bonus, $consumer_key, $consumer_email, esc_url( $this->site_url ), $php_version );
+		return sprintf( 'WP-Rocket|%s%s|%s|%s|%s|%s;', $this->plugin_version, '', $consumer_key, $consumer_email, esc_url( $this->site_url ), $php_version );
```

Comment:
Is there any reason we can't remove that `$bonus` from the sprint string altogether?
`return sprintf( 'WP-Rocket|%s|%s|%s|%s|%s;', $this->plugin_version, $consumer_key, $consumer_email, esc_url( $this->site_url ), $php_version );`